varnish/Dockerfile

```diff
@@ -5,5 +5,11 @@ COPY src/* /tmp/
 RUN /tmp/docker-install.sh \
  && rm -rvf /tmp/*
 
-EXPOSE 6081
-CMD ["/usr/local/bin/chaperone", "--user", "varnish"]
+EXPOSE 6081 6085
+
+
+HEALTHCHECK --interval=1m --timeout=3s \
+  CMD ( curl -sI http://localhost:6085 | grep -i "HTTP/1.1 401 Unauthorized" ) || exit 1
```

Comment:
I wonder if we should have a more complex health-check script here like for example: https://github.com/docker-library/healthcheck as this command checks only the dashboard.

But maybe this is enough, as it will fail if the varnish daemon is stuck. To be tested.